fuzz/Cargo.toml

```diff
@@ -11,7 +11,7 @@ rust-version = "1.56.1"
 cargo-fuzz = true
```

Comment:
Are you using this target? If not, should it be omitted?

Reply:
@xansec Not needed. I removed it.